pizza_cutter/des_pizza_cutter/integration_test/tests/test_coadding_end2end.py

```diff
@@ -469,3 +484,56 @@ def test_coadding_end2end_weight(coadd_end2end):
     wgt = m.get_cutout(0, 0, type='weight')
     nse = m.get_cutout(0, 0, type='noise')
     np.allclose(wgt, 1.0 / np.var(nse))
+
+
+def test_coadding_end2end_gaia_stars(coadd_end2end):
+    """
+    make sure pixels are getting masked
+    """
+    m = meds.MEDS(coadd_end2end['meds_path'])
+    bmask = m.get_cutout(0, 0, type='bmask')
+
+    if False:
+        import matplotlib.pyplot as plt
+        fig, axs = plt.subplots(nrows=1, ncols=1)
+        axs.imshow((bmask & BMASK_GAIA_STAR) != 0)
+        fig.savefig('/astro/u/esheldon/www/tmp/plots/tmp.png', dpi=150)
```

Comment:
```suggestion
        import pdb
        pdb.set_trace()
```

Then run this test with `pytest -vvs -x --pdb` and type `plt.show()` in the debugger.

Reply:
I've been working on this at BNL, too slow to bring up a window, but I agree this path should  not be hard coded